VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/CollisionSwitchInspector.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Visual Pinball Engine
+// Copyright (C) 2021 freezy and VPE Team
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// ReSharper disable AssignmentInConditionalExpression
+
+using UnityEditor;
+using UnityEngine;
+
+namespace VisualPinball.Unity.Editor
+{
+	[CustomEditor(typeof(CollisionSwitchComponent)), CanEditMultipleObjects]
+	public class CollisionSwitchInspector : ItemInspector
+	{
+		protected override MonoBehaviour UndoTarget => throw new System.NotImplementedException();
+
+		public override void OnInspectorGUI()
+		{
+			base.OnInspectorGUI();
+
+			if (Application.isPlaying) {
+				TableApi tableApi = TableComponent.GetComponent<Player>().TableApi;
+				CollisionSwitchApi collisionSwitchApi = tableApi.CollisionSwitch(target.name);

```

Comment:
Your `target` is the component, so you don't need to reference by name, no?

Reply:
Basically the name references are probably going away, since we don't rely on names anymore. I kept them because of potential GLE scripts that might use them, but internally we should use references by component wherever possible.

Reply:
Updated, but now it looks like this:


```
CollisionSwitchApi collisionSwitchApi = (CollisionSwitchApi)tableApi.CollisionSwitch((MonoBehaviour)target);
```

Not sure if their is any more specific cast than `MonoBehaviour`

Also we could `tableApi.Switchable` but this goes back to the other comment.

Reply:
```cs
if (switchApi is CollisionSwitchApi { IsHittable: true }) {
	DrawSwitch($"Collision switch exposed as {target.name}.", switchApi.IsSwitchEnabled);
}
```
is fine.